src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -257,6 +257,10 @@ Note que slots nomeados precisam ser um filho imediato de um componente. Você n
 Slots nomeados também podem ser passados para [componentes de frameworks de UI](/pt-br/guides/framework-components/)!
 :::
 
+:::note
+Não é possível gerar dinamicamente um slot nomeado no Astro, como por exemplo dentro de uma função map. Se essa funcionalidade é necessária dentro de um componente de um framework de interface, o melhor é gerar esses slots dinâmicos dentro do próprio framework.
```

Comment:
The other pages keep the word "UI" instead of translating it ("interface") - I would keep it for consistency's sake here.
```suggestion
Não é possível gerar dinamicamente um slot nomeado no Astro, como por exemplo dentro de uma função map. Se essa funcionalidade é necessária dentro de um componente de um framework de UI, o melhor é gerar esses slots dinâmicos dentro do próprio framework.
```